### DISCUSSION TITLE
The redirect URI 127.0.0.1 does not match configuration

### DISCUSSION THREAD

**Discussion by pavelvaskovic** [Azure-Samples/ms-identity-python-webapp#95] [Q&A] [answered]
Hi, 

I am having problem running this sample project. As I am getting AADSTS50011 error. 

![image](https://user-images.githubusercontent.com/51174473/223479485-9555e70a-74e9-4fdf-86d6-775548e613c8.png)

This is configuration of App Registration in Portal:

![image](https://user-images.githubusercontent.com/51174473/223482571-f07b7fef-845e-4766-ac22-a2a499c92ed0.png)

&
 
![image](https://user-images.githubusercontent.com/51174473/223479924-da71ff63-165e-45c2-840b-e6ced4e99193.png)

Can't figure out where is the problem. Any help would be appreciated.

Thanks!

**Comment by rayluo** [comment, marked as answer]
As suggested by the error from your first screenshot, the redirect URIs in your first two screenshots were different. Did you follow [step 6 of this document](https://learn.microsoft.com/en-us/azure/active-directory/develop/web-app-quickstart?pivots=devlang-python&tabs=windows#step-6-run-the-sample-app) to start Flask?
